airbyte-integrations/connectors/source-mysql-strict-encrypt/src/test/resources/expected_spec.json

```diff
@@ -204,6 +204,15 @@
                 "enum": ["CDC"],
                 "default": "CDC",
                 "order": 0
+              },
+              "initial_waiting_seconds": {
+                "type": "integer",
+                "title": "Initial Waiting Time in Seconds (Advanced)",
+                "description": "The amount of time the connector will wait when it launches to determine if there is new data to sync or not. Defaults to 300 seconds. Valid range: 120 seconds to 1200 seconds. Read about <a href=\"https://docs.airbyte.com/integrations/sources/mysql/#change-data-capture-cdc\">initial waiting time</a>.",
+                "default": 300,
+                "order": 4,
+                "min": 120,
```

Comment:
Not sure if 120 seconds is enough for MySQL. Previously Subodh found that 5 min is the minimum for some production MySQL instance. @subodh1810, what minimum initial waiting time would you recommend? Or should Akash do more experiments?

Reply:
Ack - I've bumped it up to 300 seconds